types/picomatch/lib/picomatch-options.d.ts

```diff
@@ -0,0 +1,175 @@
+import picomatch = require('./picomatch');
+
+export interface PicomatchOptions {
```

Comment:
Making this a separate file _does_ imply that `node_modules/picomatch/lib/picomatch-options.js` exists, though it’s not so bad since you’re not implying that any values are exported from it. Nonetheless, this could theoretically cause issues. Unfortunately in DT we aren’t at the liberty to organize files however we see fit; we have to keep a 1:1 mapping to the JS.